<ui version="4.0" >
 <class>PrintUi</class>
 <widget class="QDialog" name="PrintUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="ButtonGroup3" >
     <property name="title" >
      <string>Pages</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="1" >
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="fromED" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="toolTip" >
             <string>Page number to print from</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="toLA" >
            <property name="text" >
             <string>&amp;To:[[as in 'From page x to page y']]</string>
            </property>
            <property name="buddy" >
             <cstring>toED</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toED" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="toolTip" >
             <string>Page number to print to</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="rangeRB" >
          <property name="toolTip" >
           <string>Print all pages</string>
          </property>
          <property name="text" >
           <string>Fro&amp;m</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QRadioButton" name="allRB" >
          <property name="toolTip" >
           <string>Print all pages</string>
          </property>
          <property name="text" >
           <string>&amp;All</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="oddCB" >
        <property name="text" >
         <string>Print &amp;odd-numbered pages</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="evenCB" >
        <property name="text" >
         <string>Print &amp;even-numbered pages</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="reverseCB" >
        <property name="toolTip" >
         <string>Print in reverse order</string>
        </property>
        <property name="text" >
         <string>Re&amp;verse order</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="GroupBox3" >
     <property name="title" >
      <string>Copies</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QSpinBox" name="copiesSB" >
        <property name="toolTip" >
         <string>Number of copies</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="collateCB" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>Collate copies</string>
        </property>
        <property name="text" >
         <string>&amp;Collate</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="printPB" >
       <property name="text" >
        <string>&amp;Print</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>Close</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="ButtonGroup1" >
     <property name="title" >
      <string>Print Destination</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="printerRB" >
        <property name="toolTip" >
         <string>Send output to the printer</string>
        </property>
        <property name="text" >
         <string>P&amp;rinter:</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="printerED" >
        <property name="toolTip" >
         <string>Send output to the given printer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="fileRB" >
        <property name="toolTip" >
         <string>Send output to a file</string>
        </property>
        <property name="text" >
         <string>&amp;File:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="fileED" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="toolTip" >
         <string>Send output to a file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="browsePB" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_helpers.h</include>
 </includes>
 <tabstops>
  <tabstop>printerRB</tabstop>
  <tabstop>printerED</tabstop>
  <tabstop>fileRB</tabstop>
  <tabstop>fileED</tabstop>
  <tabstop>browsePB</tabstop>
  <tabstop>allRB</tabstop>
  <tabstop>rangeRB</tabstop>
  <tabstop>fromED</tabstop>
  <tabstop>toED</tabstop>
  <tabstop>oddCB</tabstop>
  <tabstop>evenCB</tabstop>
  <tabstop>reverseCB</tabstop>
  <tabstop>copiesSB</tabstop>
  <tabstop>collateCB</tabstop>
  <tabstop>printPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
